src/gql/generated/types.ts

```diff
@@ -104,6 +111,49 @@ export type Host = {
   uptime?: Maybe<Scalars["Time"]>;
   elapsed?: Maybe<Scalars["Time"]>;
   startedBy: Scalars["String"];
+  provider: Scalars["String"];
+  lastCommunicationTime?: Maybe<Scalars["Time"]>;
+  noExpiration: Scalars["Boolean"];
+  instanceType?: Maybe<Scalars["String"]>;
+  homeVolumeID?: Maybe<Scalars["String"]>;
+  user?: Maybe<Scalars["String"]>;
+  distro?: Maybe<DistroInfo>;
+  availabilityZone?: Maybe<Scalars["String"]>;
+  instanceTags?: Maybe<Array<Maybe<InstanceTag>>>;
+  expiration?: Maybe<Scalars["Time"]>;
+};
+
+export type HostEventLogData = {
```

Comment:
This will probably go away once everyone merges to master and resolves conflicts, but just a heads up you have some of Chaya's changes in here